x-pack/plugins/apm/public/components/app/service_overview/service_overview_instances_table/index.tsx

```diff
@@ -232,8 +232,7 @@ export function ServiceOverviewInstancesTable({ serviceName }: Props) {
     memoryUsageValue: item.memoryUsage?.value ?? 0,
   }));
 
-  const isLoading =
-    status === FETCH_STATUS.LOADING || status === FETCH_STATUS.PENDING;
+  const isLoading = status === FETCH_STATUS.LOADING;
```

Comment:
I'm not sure if it's that simple. The reason why it also checks for PENDING, is because a component might be waiting on other data to come in before it starts a request. Removing a check for PENDING might mean that a component will flash "No items" before it starts displaying a loading indicator.

Reply:
and the change is not needed anymore.

Reply:
Do you mean that this change will be removed? (I still see this + other places where the `isLoading` check was changed0

Reply:
I'm going to put it back.